docs/src/examples/components/Attachment/Slots/index.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react'
+import ComponentExample from 'docs/src/components/ComponentDoc/ComponentExample'
+import ExampleSection from 'docs/src/components/ComponentDoc/ExampleSection'
+
+const Slots = () => (
+  <ExampleSection title="Slots">
+    <ComponentExample
+      title="Icon"
+      description="An attachment can have an icon."
+      examplePath="components/Attachment/Slots/AttachmentIconExample"
+    />
+    <ComponentExample
+      title="Header"
```

Comment:
header is advertised as slot, and maybe I've made a false conclusion, but the first thing I've tried was the following - essentially, one of those things that we might expect from the shorthands:

```jsx
<Attachment header={{ content: '...', styles: { color: 'red' } }} />
```

Apparently, it doesn't work. So, the question is the following: do we see this general pattern (that is applicable for shorthands) to be applicable to any slot content? Actually, more interested about our vision on that. 

To me it seems to be a very good thing to ensure this consistency in our customization mechanisms - especially given that the following approach for style tweaking seems to be deprecated:

```jsx
<Attachment styles={{ header: {..<header styles>..} }}
```

Reply:
@kuzhelov will this PR solve that? https://github.com/stardust-ui/react/pull/238

Reply:
@brianespinosa, sorry for the late response, my bad :( my promise to be better next time.

> @kuzhelov will this PR solve that? #238 

Unfortunately, now - and the reason is that `header` is currently introduced as a simple `string`, and not a Stardust component that encapsulates this knowledge on how this styling should be applied. 

However, this is quite easy to fix - we could use `Text` component for `header` and `content`, so that all functionality of slots (where main aspect is styling) will be preserved.